s4e-backend/src/main/java/pl/cyfronet/s4e/data/repository/query/PreparedStatementBuilder.java

```diff
@@ -26,6 +26,19 @@ public PreparedStatement preparedStatement(Connection connection,
         return ps;
     }
 
+    public String prepareCountQueryAndParameters(Map<String, Object> params,
```

Comment:
I'd give it private access, analogously to ::prepareQueryAndParameters method and call it in an analog of the ::preparedStatement method.
It's inconsistent now.